java/kafka/streams/src/main/java/KafkaStreamsExample.java

```diff
@@ -46,7 +46,7 @@ public static void main(String[] args) {
                 KafkaClientSupplier supplier = new TracingKafkaClientSupplier();
                 streams = new KafkaStreams(builder.build(), props, supplier);
             } else {
-                throw new RuntimeException("Error: TRACING_SYSTEM " + tracingSystem + " is not recognized or supported!");
+                throw new RuntimeException("Error: STRIMZI_TRACING_SYSTEM " + tracingSystem + " is not recognized or supported!");
```

Comment:
Why does the OPENTELEMETRY branch above on which I cannot comment configure its own SERDEs? Shouldn't that be removed?

Reply:
Yes. They were hard coded and in the yaml. Removed now.